asn1/Stream.asn

```diff
@@ -67,14 +68,46 @@ FrameSet FRAME ::= {
   {19 StreamMoneyBlocked} |
   {20 StreamData} |
   {21 StreamMaxData} |
-  {22 StreamDataBlocked}
+  {22 StreamDataBlocked} |
+  {23 StreamReceipt}
 }
 
 StreamFrame ::= SEQUENCE {
   type FRAME.&typeId ({FrameSet}),
   data FRAME.&Type ({FrameSet}{@type})
 }
 
+RECEIPT ::= CLASS {
```

Comment:
This is a minor nit, but can we move these definitions to the bottom of this file? As they are now, they sort of break the flow of the STREAM and frame definitions themselves, which makes the ASN harder to read.

We could even put these directly above the `StreamReceipt` frame definition on line 209? Or pull the receipt definitions themselves into a separate file? (I can't think of a use-case, but I suppose lines 80-109 can technically stand on their own independent of STREAM and be imported into this file).

Reply:
Moved to a separate file https://github.com/interledger/rfcs/pull/570/commits/7dd4111d08c640466d7d3aac82a0ec3f1658d0db
(force pushed because the commit wasn't originally appearing in the pull request, likely due to earlier github incident)